src/service/viewer-impl.js

```diff
@@ -130,6 +130,9 @@ export class Viewer {
     /** @private {!Observable<!JSONType>} */
     this.broadcastObservable_ = new Observable();
 
+    /** @private {!Observable<!JSONType>} */
+    this.scrollDocObservable_ = new Observable();
```

Comment:
Does it make sense to introduce more API dependencies when we are in the process of moving them out? Let's just create a generic observable for all types of messages?